code/modules/mining/mine_items.dm

```diff
@@ -249,4 +249,50 @@
 	if(colorindex >= 6)
 		colorindex = 0
 	icon_state = "mobcap[colorindex]"
-	update_icon()
\ No newline at end of file
+	update_icon()
+
+//Fans
+/obj/structure/fans
+	icon = 'icons/obj/lavaland/survival_pod.dmi'
+	icon_state = "fans"
+	name = "environmental regulation system"
+	desc = "A large machine releasing a constant gust of air."
+	anchored = 1
+	density = 1
+	var/arbitraryatmosblockingvar = 1
```

Comment:
so we can have fans that *don't* block atmos, without having an entirely separate object, silly.